### ISSUE TITLE
MD004 can't set to asterick

### ISSUE THREAD

**Issue by marklester** [DavidAnson/markdownlint#335]
```yaml
MD004:
  style: asterick
```
error 
```
MD004/ul-style Unordered list style [Expected: asterick; Actual: asterisk]
```

**Comment by DavidAnson** [comment]
Please show the Markdown text that causes the error.

**Comment by DavidAnson** [comment]
Wait, do you realize "asterisk" is spelled wrong above? That's likely the problem - invalid configuration.

**Comment by marklester** [comment]
wow how did I not see c 🤦 Sorry about that.
